base/test-core/src/test/java/org/openscience/cdk/tools/manipulator/BondManipulatorTest.java

```diff
@@ -191,6 +191,14 @@ void testDestroyBondOrder_IBond_Order() {
         Assertions.assertEquals(4.0, BondManipulator.destroyBondOrder(Order.QUADRUPLE), 0.00001);
         Assertions.assertEquals(5.0, BondManipulator.destroyBondOrder(Order.QUINTUPLE), 0.00001);
         Assertions.assertEquals(6.0, BondManipulator.destroyBondOrder(Order.SEXTUPLE), 0.00001);
+
+        // above code is deprecated in favor of this:
+        Assertions.assertEquals(1.0, Order.SINGLE.numeric(), 0.00001);
```

Comment:
Don't these return integers?

Reply:
If not now this is likely to flag something at some point

Reply:
Ah, good point. Sorry about that; still very tired it seems :( I will fix this!